wiki/Beatmap_ranking_procedure/fr.md

```diff
@@ -1,52 +1,43 @@
 
-Les [beatmaps](/wiki/Beatmap) peuvent être promu auprès de la communauté en suivant la procédure de classement de la beatmap et en entrant dans la catégorie [Ranked](/wiki/Beatmap/Category#ranked).
+Les [beatmaps](/wiki/Beatmap) peuvent être promus dans la communauté en suivant la procédure de classement des beatmaps et en entrant dans la catégorie [classé](/wiki/Beatmap/Category#ranked).
 
 ## Commentaires
 
-Le créateur d'un beatmap peut marquer ses soumissions comme `Work in Progress` ou `Pending`. Les Beatmaps dans ces deux catégories sont destinés à recevoir les commentaires de la communauté osu!
+Le créateur d'un beatmap peut marquer ses soumissions comme `Work in Progress` ou `En attente`. Les beatmaps de ces deux catégories sont destinés à recevoir les commentaires de la communauté d'osu!
 
-Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiques constructives sur une map de pointage et qui est bénéfique à l'amélioration de sa qualité. Les utilisateurs affichent habituellement leurs commentaires sur la page de discussion d'une map ou discutent directement des problèmes avec le créateur de la map.
+Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiques constructives sur un beatmap et qui permet d'en améliorer la qualité. Les utilisateurs publient généralement leurs commentaires sur la page de discussion d'une beatmap ou discutent directement avec le créateur de la beatmap.
 
-Les beatmappers les plus récents ont généralement besoin de modifications importantes pour créer des beatmaps qui répondent aux normes de qualité de la catégorie Ranked, et il leur est souvent demandé de recréer leurs beatmaps à partir de zéro.
+Les nouveaux beatmappers ont généralement besoin d'un important travail de modding pour créer des beatmaps qui répondent aux normes de qualité de la catégorie classée, et il leur est souvent demandé de recréer leurs beatmaps à partir de zéro.
 
-Avant d'être nominés pour le classement, les beatmaps doivent recevoir [hype](/wiki/Glossary#hype) des messages d'au moins cinq utilisateurs différents.
+Avant d'être proposés au classement, les beatmaps doivent recevoir des messages de [hype](/wiki/Glossary#hype) d'au moins cinq utilisateurs différents.
 
 ## Nominations
 
-Les membres des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des modders expérimentés qui sont capables de nommer des beatmaps pour le classement. Leur rôle est de s'assurer qu'une beatmap répond aux critères du [Ranking Criteria](/wiki/Ranking_Criteria) ainsi que leurs propres normes de qualité personnelles.
+Les membres du groupe des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des moddeurs expérimentés capables de proposer des beatmaps classable. Leur rôle est de s'assurer qu'une beatmapset répond aux [critères de classement](/wiki/Ranking_Criteria) ainsi qu'à leurs propres critères de qualité.
```

Comment:
please replace `beatmapset` with an equivalent of `beatmap` in French as I did recently